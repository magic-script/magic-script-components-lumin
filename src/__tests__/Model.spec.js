//
import React from "react";
import { create } from "react-test-renderer";
import { Model } from "magic-script-components";

describe("Model component", () => {
  test("Matches the snapshot", () => {
    const model = create(
      <Model modelPath="" materialPath="" texturePath="" textureName="" />
    );
    expect(model.toJSON()).toMatchSnapshot();
  });
});
