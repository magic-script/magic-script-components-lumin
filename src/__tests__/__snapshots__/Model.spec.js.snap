// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Model component Matches the snapshot 1`] = `
<model
  materialPath=""
  modelPath=""
  textureName=""
  texturePath=""
/>
`;
